src/TestProblem/Hydro/ClusterMerger/Init_TestProb_ClusterMerger.cpp

```diff
@@ -342,6 +475,20 @@ void SetParameter()
 
       } // if ( Merger_Coll_NumHalos > 2 && Merger_Coll_IsGas3 )
 
+//    (2-2) Initialize the BH position and velocity
+      double ClusterCenter[3][3] = {{ Merger_Coll_PosX1, Merger_Coll_PosY1, amr->BoxCenter[2] },
+                                    { Merger_Coll_PosX2, Merger_Coll_PosY2, amr->BoxCenter[2] },
+                                    { Merger_Coll_PosX3, Merger_Coll_PosY3, amr->BoxCenter[2] }};
+      double CenterVel[3][3]     = {{ Merger_Coll_VelX1, Merger_Coll_VelY1, 0.0 },
```

Comment:
Strictly speaking, we should allow for the z-axis position and velocity to be different from the center and 0.